bottlerocket-settings-models/modeled-types/src/kubernetes.rs

```diff
@@ -353,14 +358,14 @@ mod test_kubernetes_cluster_name {
 
     #[test]
     fn good_cluster_names() {
-        for ok in &["more-chars_here.now", &"a".repeat(63)] {
+        for ok in &["more-chars_here-123", &"a".repeat(100)] {
```

Comment:
nit: maybe a couple more test cases since we now allow trailing dashes and underscores:
```suggestion
        for ok in &["more-chars_here-123",  "trailing-dash-", "under_score_", &"a".repeat(100)] {
```

Reply:
@bcressey done!